spec/inputs/jdbc_spec.rb

```diff
@@ -113,4 +114,59 @@
 
     @database.drop_table(:test_table)
   end
+
+  context "persistence" do
+
+    it "should respect last run metadata" do
```

Comment:
I was hoping to use some of these across tests, but `:settings` is different in almost every `it` block.